packages/core/src/pubsub/pubsub.ts

```diff
@@ -24,7 +24,8 @@ function ipfsToPubsub(
       of(incoming).pipe(
         map((incoming) => {
           const message = deserialize(incoming);
-          const logMessage = { ...incoming, data: message };
+          const serializedMessage = serialize(message);
```

Comment:
do we need the call to serialize() given that `message` came from calling deserialize() on `incoming`?  Can we just use `incoming` directly?

Reply:
Tried it and no. Incoming is a different format